examples/notebooks/beam-ml/data_preprocessing/huggingface_text_embeddings.ipynb

```diff
@@ -0,0 +1,456 @@
+            "Embedding shape: 10\n",
+            "{'x': [-0.0343877375125885, 0.07250142097473145, 0.01443990133702755, -0.03669498860836029, 0.014018685556948185, 0.06307007372379303, 0.03468254581093788, -0.014530746266245842, -0.05986189469695091, -0.04538322612643242]}\n",
+            "Embedding shape: 10\n",
+            "{'x': [-0.005963834468275309, 0.025043703615665436, -0.003182061715051532, -0.025242920964956284, -0.0398230254650116, -0.012771873734891415, 0.0447133406996727, 0.014535333029925823, -0.03821341320872307, -0.04114910215139389]}\n",
+            "Embedding shape: 10\n",
+            "{'x': [-0.039007965475320816, -0.010609461925923824, -0.007382705342024565, -0.050189778208732605, -0.0025175788905471563, -0.0416409894824028, 0.02696940489113331, -0.014800631441175938, -0.014126974157989025, -0.061636749655008316]}\n",
+            "Embedding shape: 10\n",
+            "{'x': [-0.09598278254270554, -0.06301165372133255, -0.11690578609704971, -0.05907457321882248, -0.05132286250591278, -0.0034391973167657852, 0.018687350675463676, 0.006543711293488741, -0.04905705526471138, -0.031649429351091385]}\n",
+            "Embedding shape: 10\n",
+            "{'x': [-0.011600406840443611, 0.05651004612445831, 0.016623979434370995, -0.09469003975391388, -0.009865491650998592, 0.07234735041856766, 0.04412448778748512, -0.0411749929189682, -0.04212445020675659, -0.10263106226921082]}\n",
+            "Embedding shape: 10\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Pass additional arguments that are supported by `sentence-transformer` models, such as `convert_to_numpy=False`. These arguments are passed as a `dict` to the `SentenceTransformerEmbeddings` transform by using the `inference_args` parameter.\n",
+        "\n",
+        "By passing `convert_to_numpy=False`, the output will contain `torch.Tensor`s."
+      ],
+      "metadata": {
+        "id": "1MFom0PW_vRv"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "artifact_location_minilm_with_inference_args = tempfile.mkdtemp(prefix='huggingface_')\n",
+        "\n",
+        "embedding_transform = SentenceTransformerEmbeddings(\n",
+        "        model_name=text_embedding_model_name, columns=['x'],\n",
+        "        inference_args={'convert_to_numpy': False}\n",
+        "        )\n",
+        "\n",
+        "with beam.Pipeline() as pipeline:\n",
+        "  data_pcoll = (\n",
+        "          pipeline\n",
+        "          | \"CreateData\" >> beam.Create(content))\n",
+        "  transformed_pcoll = (\n",
+        "      data_pcoll\n",
+        "      | \"MLTransform\" >> MLTransform(write_artifact_location=artifact_location_minilm_with_inference_args).with_transform(embedding_transform))\n",
```

Comment:
Nit: the formatting is a little funky here, this level of tabbing should match the expression right above it

Reply:
Same comment applies elsewhere